Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4837,8 +4847,7 @@ def __init__(self):
             self.http_client_print = getattr(http_client, 'print', None)  # save in case someone else patched it already
             setattr(http_client, 'print', self.int_logger.print_override)
         self.handler = DemistoHandler()
```

Comment:
There is no need for the DemistoFormatter. The fix should be that the handler should use the IntegrationLogger to write to the log and not directly to `demisto.debug()`. I think the following will fix the issue:
```suggestion
        self.handler = DemistoHandler(self.int_logger)
```